### DISCUSSION TITLE
Project Setup

### DISCUSSION THREAD

**Discussion by wintercodeNKJJ** [PokeAPI/pokeapi#906] [Q&A] [answered]
`make setup` command does not function on my computer. I get the following error.
```bash
make: *** [Makefile:19: setup] Error -1073741515
```

how do I go about it?

**Comment by Naramsim** [comment]
Hmm, does it output more lines? Can you check what's the content of 'make setup' and run it without make?

**Reply by wintercodeNKJJ** [reply]
> does it output more lines? 

NO just that line
![Capture](https://github.com/PokeAPI/pokeapi/assets/121441148/40522714-f6cf-4792-a4fc-239e19c5747d)

> Can you check what's the content of 'make setup'

Makefile line 1 to 38
![image](https://github.com/PokeAPI/pokeapi/assets/121441148/3bedec5c-e50b-431a-978b-e629e4f0ff54)

>run it without make?

```bash
python manage.py migrate --settings=config.local
```
![Capture2](https://github.com/PokeAPI/pokeapi/assets/121441148/28724a4c-1c52-4b5e-b485-ce8e1efb2d85)

**Reply by Naramsim** [reply, marked as answer]
Hi, I think it's a local problem with your make. it's also strange that the `python` command doesn't output anything.
